tracing-subscriber/src/layer.rs

```diff
@@ -586,9 +586,24 @@ pub struct Identity {
 /// [`Context::scope`]: struct.Context.html#method.scope
 #[cfg(feature = "registry")]
 #[cfg_attr(docsrs, doc(cfg(feature = "registry")))]
-pub struct Scope<'a, L: LookupSpan<'a>>(
-    Option<std::iter::Chain<registry::FromRoot<'a, L>, std::iter::Once<SpanRef<'a, L>>>>,
-);
+#[deprecated(note = "moved to crate::registry::ScopeFromRoot")]
```

Comment:
nit: let's add the version here (and maybe `s/moved/renamed`)?
```suggestion
#[deprecated(
    note = "renamed to crate::registry::ScopeFromRoot",
    since = "0.2.19",
)]
```